frontend/tests/shared/utils/helpers.test.js

```diff
@@ -1,4 +1,5 @@
 import {addOuterTag} from "shared/utils/_helpers";
+import helper from "shared/utils/helpers";
```

Comment:
I saw the CI error; you may need to add this to the `_helpers` file since I think the JS testing framework doesnt have access to a browser, and therefore doesnt have the external jQuery that we vendor in... @casey1173

Reply:
ah okay will do, thanks! Got confused by that one